packages/auto_mappr/lib/src/extensions/expression_extension.dart

```diff
@@ -120,4 +120,6 @@ extension ExpressionExtension on Expression {
   Expression nullabled() {
     return refer('${accept(DartEmitter())}?');
   }
+
+  Expression equalToNull() => equalTo(refer('null'));
```

Comment:
use `literalNull` instead of `refer('null')`?

Reply:
Fixed on every place i've found it